<launch>
	<include file="$(find lasr_perception_server)/launch/perception_server.launch"/>
	<rosparam command="load" file="$(find coffee_shop)/config/test_wait_for_person.yaml" />
	<node pkg="coffee_shop" type="test_wait_for_person.py" name="test_wait_for_person" output="screen"/>
</launch>
